trade-finance-manager-ui/server/helpers/deal-cancellation-enabled.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as ukefCommon from '@ukef/dtfs2-common';
+import { DEAL_SUBMISSION_TYPE } from '@ukef/dtfs2-common';
+import { dealCancellationEnabled } from './deal-cancellation-enabled';
+
+describe('dealCancellationEnabled', () => {
+  const isTfmDealCancellationFeatureFlagEnabledSpy = jest.spyOn(ukefCommon, 'isTfmDealCancellationFeatureFlagEnabled');
+
+  describe('when TFM Deal Cancellation feature flag is enabled', () => {
+    beforeEach(() => {
+      isTfmDealCancellationFeatureFlagEnabledSpy.mockReturnValue(true);
+    });
+
+    it('should return false if the deal type is MIA', () => {
+      const result = dealCancellationEnabled(DEAL_SUBMISSION_TYPE.MIA);
+
+      expect(result).toEqual(false);
+    });
+
+    it('should return true if the deal type is MIN', () => {
+      const result = dealCancellationEnabled(DEAL_SUBMISSION_TYPE.MIN);
+
```

Comment:
Do you know how to resolve these TS errors?

Reply:
I am not seeing a ts error?

Reply:
It's Codacy being broken in TS :(

Reply:
ooh I'll take a look if its a genuine thing, just so used to codacy i assumed that was it :(

Reply:
Yeah looks fine locally for me - probably just ts/eslint server not recognising a new file has been made